web/ui/react-app/src/App.tsx

```diff
@@ -69,25 +69,45 @@ const App: FC<AppProps> = ({ consolesLink }) => {
     >
       <Theme />
       <PathPrefixContext.Provider value={basePath}>
-        <Navigation consolesLink={consolesLink} />
-        <Container fluid style={{ paddingTop: 70 }}>
-          <Router basepath={`${basePath}`}>
-            <Redirect from="/" to={`graph`} noThrow />
-            {/*
+        <Router basename={basePath}>
+          <Navigation consolesLink={consolesLink} />
+          <Container fluid style={{ paddingTop: 70 }}>
```

Comment:
probably not a big deal, but just for sanity, you switch the `Container` position. Before it was encapsulating the `Router` . Now it's the opposite.

Reply:
Yeah, I had to wrap the `Router` around the `Navigation` now too because the `Navigation` contains `Link`s and those are only valid within a `Router` in React Router (it works a bit differently).